server/src/main/scala/io/delta/sharing/server/DeltaSharingService.scala

```diff
@@ -240,12 +240,28 @@ class DeltaSharingService(serverConfig: ServerConfig) {
   }
 
   @Head("/shares/{share}/schemas/{schema}/tables/{table}")
+  @Get("/shares/{share}/schemas/{schema}/tables/{table}")
```

Comment:
It seems that there's no way to "not support startingTimestamp for HEAD" given this config.

Reply:
Does this imply that oss server will support startingTimestamp for HEAD method but DB's server will not?

Reply:
Yes..

Reply:
But we can mention in the protocol doc that startingTimestamp is not supported for HEAD